spec/level_db_spec.rb

```diff
@@ -65,6 +65,50 @@
       db.close
     end
 
+    describe '#suspend_compactions' do
+      it 'suspends compactions' do
+        db = double(:db)
+        db.stub(:suspend_compactions)
+        LevelDb::Db.new(db).suspend_compactions
+        db.should have_received(:suspend_compactions)
+      end
+    end
+
+    describe '#resume_compactions' do
+      it 'resumes compactions' do
+        db = double(:db)
+        db.stub(:resume_compactions)
+        LevelDb::Db.new(db).resume_compactions
+        db.should have_received(:resume_compactions)
+      end
+    end
+
+    describe '#compact_range' do
+      let :leveldb do
+        double(:db, compact_range: nil)
+      end
+
+      it 'compacts the given range' do
+        LevelDb::Db.new(leveldb).compact_range(from: 'one', to: 'three')
+        leveldb.should have_received(:compact_range).with('one'.to_java_bytes, 'three'.to_java_bytes)
+      end
+
+      it "is ok to omit the 'from' option" do
+        LevelDb::Db.new(leveldb).compact_range(to: 'three')
+        leveldb.should have_received(:compact_range).with(nil, 'three'.to_java_bytes)
+      end
+
+      it "is ok to omit the 'to' option" do
+        LevelDb::Db.new(leveldb).compact_range(from: 'one')
+        leveldb.should have_received(:compact_range).with('one'.to_java_bytes, nil)
+      end
+
+      it "is ok to omit all options" do
+        LevelDb::Db.new(leveldb).compact_range
+        leveldb.should have_received(:compact_range).with(nil, nil)
+      end
```

Comment:
I really don't like these tests, but I couldn't really think of any way to test the compactions.